.config.sample.env

```diff
@@ -1,99 +0,0 @@
-export BOOTSTRAP_FLUX_GITHUB_WEBHOOK_SECRET="generated" # NOTE: Must only contain alphanumeric characters and dashes
-
-# The Weave GitOps dashboard admin password
-# `generated` - this will generate a token and print it in the logs
-# Set this to any other string and it will be used for the secret
-export BOOTSTRAP_WEAVE_GITOPS_ADMIN_PASSWORD="generated" # NOTE: Must only contain alphanumeric characters and dashes
-
-# The Grafana admin password
-# `generated` - this will generate a token and print it in the logs
-# Set this to any other string and it will be used for the secret
-export BOOTSTRAP_GRAFANA_ADMIN_PASSWORD="generated" # NOTE: Must only contain alphanumeric characters and dashes
-
-# Age Public Key - string should start with age
-# e.g. age15uzrw396e67z9wdzsxzdk7ka0g2gr3l460e0slaea563zll3hdfqwqxdta
-export BOOTSTRAP_AGE_PUBLIC_KEY=""
-
-# Choose your timezone
-# e.g. America/New_York
-export BOOTSTRAP_TIMEZONE="Etc/UTC"
-
-# The email you want to be associated with the ACME account
-export BOOTSTRAP_ACME_EMAIL=""
-
-# Choose one of your cloudflare domains
-# e.g. onedr0p.com
-export BOOTSTRAP_CLOUDFLARE_DOMAIN=""
-# Your global Cloudflare API Token (not API Key)
-export BOOTSTRAP_CLOUDFLARE_APITOKEN=""
-
-# Create a cloudflare tunnel to automatically ingress into your cluster
-# https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/
-# cloudflared tunnel create <NAME or UUID>
-export BOOTSTRAP_CLOUDFLARE_ACCOUNT_TAG=""
-export BOOTSTRAP_CLOUDFLARE_TUNNEL_SECRET=""
-export BOOTSTRAP_CLOUDFLARE_TUNNEL_ID=""
-
-# The load balancer IP for k8s_gateway, choose a available IP in your nodes network that is not being used
-export BOOTSTRAP_K8S_GATEWAY_ADDR=""
-# The load balancer IP for ingress-nginx, choose a available IP in your nodes network that is not being used
-export BOOTSTRAP_INGRESS_NGINX_ADDR=""
-# The IP address to use with kube-vip, choose a available IP in your nodes network that is not being used
-export BOOTSTRAP_KUBE_VIP_ADDR=""
-# Choose your cluster or service cidrs
-# Leave this unchanged unless you know what you are doing
-export BOOTSTRAP_CLUSTER_CIDR="10.42.0.0/16"
-export BOOTSTRAP_SERVICE_CIDR="10.43.0.0/16"
-
-#
-# Ansible related variables
-#
-
-#
-# Default prefixes for hostnames assigned by Ansible
-# These are unused on nodes where BOOTSTRAP_ANSIBLE_HOSTNAME_ is provided
-#
-
-export BOOTSTRAP_ANSIBLE_DEFAULT_CONTROL_NODE_HOSTNAME_PREFIX="k8s-" # NOTE: Must only contain alphanumeric characters and dashes
-export BOOTSTRAP_ANSIBLE_DEFAULT_NODE_HOSTNAME_PREFIX="k8s-" # NOTE: Must only contain alphanumeric characters and dashes
-
-#
-# Ansible hosts - repeat this block as many times as you need,
-# incrementing the last digit on the variable name for each node
-#
-
-# IP Address to the node
-# That doesn't conflict with any other IP addresses here
-# e.g. 192.168.1.200
-export BOOTSTRAP_ANSIBLE_HOST_ADDR_0=""
```

Comment:
Do we need this somewhere if there are specific users per host?

Reply:
This is handled by the inventory.